src/types/apps.ts

```diff
@@ -190,9 +190,9 @@ export type AutocompleteDynamicSelect = AutocompleteElement & {
     call: AppCall;
 };
 
-export type AutocompleteUserSelect = AutocompleteElement & {}
+export type AutocompleteUserSelect = AutocompleteElement & unknown
 
-export type AutocompleteChannelSelect = AutocompleteElement & {}
+export type AutocompleteChannelSelect = AutocompleteElement & unknown
```

Comment:
I'm thinking this is what should be

```suggestion
export type AutocompleteChannelSelect = AutocompleteElement;
```

Reply:
Good point. The code is more readable that way and it the same anyway.